site/docs/configuration/reference.md

```diff
@@ -64,14 +64,115 @@ A test case represents a single example input that is fed into all prompts and p
-| value            | string | No       | The expected value, if applicable                                                                                                                                                                                                           |
-| threshold        | number | No       | The threshold value, applicable only to certain types such as `similar`, `cost`, `javascript`, `python`                                                                                                                                     |
-| provider         | string | No       | Some assertions (type = similar, llm-rubric, model-graded-\*) require an [LLM provider](/docs/providers)                                                                                                                                    |
-| metric           | string | No       | The label for this result. Assertions with the same `metric` will be aggregated together                                                                                                                                                    |
-| contextTransform | string | No       | JavaScript expression to extract context from provider output for context-based assertions. Alternative to providing context as a test variable. Supports expressions like `output.context` or `output.docs.map(d => d.content).join('\n')` |
+| Property         | Type   | Required | Description                                                                                                                                             |
+| ---------------- | ------ | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| type             | string | Yes      | Type of assertion                                                                                                                                       |
+| value            | string | No       | The expected value, if applicable                                                                                                                       |
+| threshold        | number | No       | The threshold value, applicable only to certain types such as `similar`, `cost`, `javascript`, `python`                                                 |
+| provider         | string | No       | Some assertions (type = similar, llm-rubric, model-graded-\*) require an [LLM provider](/docs/providers)                                                |
+| metric           | string | No       | The label for this result. Assertions with the same `metric` will be aggregated together                                                                |
+| contextTransform | string | No       | Javascript expression to dynamically construct context for context-based assertions (`context-faithfulness`, `context-relevance`, and `context-recall`) |
+
+#### Context Transform
```

Comment:
this is far too much in this file. we should edit it very sparingly. find a better place for this